README.md

```diff
@@ -8,6 +8,10 @@ Generates a Typescript type definition file for a C# POCO class.  Takes in a str
 - Gulp: https://github.com/ffMathy/gulp-typescript-cs-poco
 - Grunt: https://github.com/ffMathy/grunt-typescript-cs-poco
 
+## Installation with npm
+
+`npm install --save typescript-cs-poco`
```

Comment:
I do not think so. I'm developing a tool for compiling online, and in my case I'll use it in production.

How about considering `npm install typescript-cs-poco`.